.github/workflows/curseforge_release.yml

```diff
@@ -0,0 +1,42 @@
+# Requires a Release on GitHub of the current version
+name: Release to CurseForge
+
+# manually run, or automatically after the GitHub release is done
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: [ "Release to GitHub" ]
+    types:
+      - completed
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Get Mod Version
+        id: get_version
+        run: echo ::set-output name=version::$(./gradlew getVersionFromJava --q)
```

Comment:
Right now we append `beta` to the mod jar names, will this step keep the `beta` in the jar names?

Reply:
This uses the same function as the one which determines the jar name. Since the jar has the `beta` suffix, so will the release title for both Curseforge and GitHub.